package.json

```diff
@@ -59,7 +59,7 @@
     "injection"
   ],
   "dependencies": {
-    "which": "^3.0.0"
+    "which": "^3.0.0 || ^4.0.0 || ^5.0.0"
```

Comment:
I'm not actually sure this is allowed and works in practice, but I'm willing to give it a try. If, after publishing, it turns out the answer is "no" it should be easy enough to publish a hotfix.

Just need to remember to test it myself so that if it doesn't work I find out sooner rather than later.